scripts/Inelastic/Direct/PropertyManager.py

```diff
@@ -607,11 +607,10 @@ def find_files_to_sum(self,num_files=None):
 
         """
         # this returns only runs, left to sum with current sample_run sum settings
-#pylint: disable=unused-variable
-        runs,sum_ws,added      = PropertyManager.sample_run.get_runs_to_sum(None,num_files)
+        runs,_,_      = PropertyManager.sample_run.get_runs_to_sum(None,num_files)
```

Comment:
Whitespace

Reply:
fixed but the system tests were hanging up despite all fine on my local machine and the code is not so different from the previous one. 

Need thorough investigation with @martyngigg and I have no time to do it at the moment.